test/generate_parse_fixture_yml.py

```diff
@@ -3,32 +3,40 @@
 
 import oyaml as yaml
 
-from sqlfluff.core.parser import Parser, Lexer
-from sqlfluff.core import FluffConfig
 from sqlfluff.cli.commands import quoted_presenter
 
-from dialects.parse_fixtures import get_parse_fixtures, load_file
+from conftest import compute_parse_tree_hash, parse_example_file
+from dialects.parse_fixtures import get_parse_fixtures
 
 yaml.add_representer(str, quoted_presenter)
 
-parse_success_examples, _ = get_parse_fixtures()
```

Comment:
I refactored this script, splitting it into functions and moving some reusable code to `conftest.py`, where it is also used by `test/dialects/dialects_test.py::test__dialect__base_parse_struct`.